### ISSUE TITLE
scroll too low when data > 1000 on Android

### ISSUE THREAD

**Issue by vivianking6855** [sunnylqm/react-native-alphabetlistview#16]
scroll too low when data > 1000  Android, if there any optimization for this critical issue.

**Comment by joshbuchea** [comment]
@vivianking6855 you can set the initialListSize={INGREDIENTS.length}

**Comment by joshbuchea** [comment]
@vivianking6855 oops accidentally posted that without finishing my comment. If you set the `initialListSize` prop to the total number of rows for your list, it will render the entire list at once.

**Comment by vivianking6855** [comment]
yes, it works well when initialListSize<100, otherwise there will be more than 2s for UI loading

**Comment by joshbuchea** [comment]
@vivianking6855 yes, I'm seeing similar performance with a large data set (~ 750 items).

**Comment by vivianking6855** [comment]
I got SG Lsit view onanother source： https://github.com/sghiassy/react-native-sglistview
I may help us to optimize ListView performance

**Comment by rgovindji** [comment]
You can checkout my implementation here which handles larger lists https://github.com/rgovindji/react-native-atoz-list

This uses the code from https://github.com/brentvatne/fixed-height-windowed-list-view-experiment but just packaged up a little more user friendly. I'll try to clean it up more and add it to npm in a few days.

**Comment by niftylettuce** [comment]
@rgovindji your solution has many bugs, and is not stable with the latest RN release

**Comment by niftylettuce** [comment]
also yes, this is extremely slow with even 500+ contacts

**Comment by roshangm1** [comment]
I still have this issue. I have around 3500+ data. All the data are not loaded at once which is causing me issue to quickly jump to the data at last. It works fine for some data starting with A. 
Please help

**Comment by sibelius** [comment]
FlatList could solve this problems

**Comment by roshangm1** [comment]
@sibelius this actually works with normal listview. Maybe, listview is implemented in such a way with this library ? Noob here.

**Comment by germanurrus** [comment]
Is there any updates about this performance issue? I tried to set initialListSize (I'm using a fixed list of 300 elements) and this makes application to take time to respond to any touch events....it's like it's processing the elements, and then the list works perfectly...but the downside is the time it takes for the application to responde at start time...any ideas how to overcome this issue? thanks!
